source/features/vertical-front-matter.tsx

```diff
@@ -9,15 +9,24 @@ import features from '.';
 const hasFrontMatter = (): boolean => pageDetect.isSingleFile() && /\.(mdx?|mkdn?|mdwn|mdown|markdown|litcoffee)$/.test(location.pathname);
 
 function init(): void | false {
-	const table = select('[data-table-type="yaml-metadata"]')!;
+	const table = select('.markdown-body > table:first-child');
```

Comment:
This will create issues, but hopefully first-child 2-rows tables are rare enough 😬

Reply:
That would still be something like key-value pairs, so the direction change won't be confusing.

Reply:
True, but if someone expect a horizontal table and sees a vertical table it can still cause confusion